node-tests/acceptance/build-test.js

```diff
@@ -73,10 +73,7 @@ describe('Acceptance', function() {
         expect(output.manifest()).to.deep.equal(expectedManifests['lazy']);
         output.contains('assets/node-asset-manifest.js');
         // Engine inherites its environment from the parent
-        output.contains(
-          `engines-dist/${engineName}/config/environment.js`,
-          /"environment":"development"/
```

Comment:
Why would we want this assertion removed?

Reply:
@rwjblue the old solution has a problem where the `environment` always returns the` development` value. And that test just covered that scenario, I preferred to make sure that only the `config/environment` is exposed out.

But I agree we should test all scenarios.

Reply:
yes, we will need to test more then one scenario

Reply:
I'll do this between today and tomorrow.